### PR TITLE
hot fixe disable registration

### PR DESCRIPTION
**Pull request by Mh-Asmi**
This pull request makes the following changes:
- stop post title text validation
- fix registration disable

Test checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: src/Ushahidi/Modules/V5/Actions/Auth/Handlers/RegisterCommandHandler.php [hunk 1]
```diff
@@ -30,6 +34,16 @@ public function __invoke(Action $action)
          * @var RegisterCommand $action
          */
         $this->isSupported($action);
+        $this->checkDisapleRegisteration();
```
Comment (tuxpiper) [inline src/Ushahidi/Modules/V5/Actions/Auth/Handlers/RegisterCommandHandler.php:37]: Just a small typo here @Mh-Asmi  . "Disaple" should be "Disable"

**Review by tuxpiper — COMMENTED** [review]
All good for me, just a small typo

**Review by tuxpiper — APPROVED** [review]

**Comment by tuxpiper** [comment]
ah @Mh-Asmi if it's a hot fix, can we merge to the release branch instead?

**Comment by Mh-Asmi** [comment]
> ah @Mh-Asmi if it's a hot fix, can we merge to the release branch instead?

will we merge "release/2023.32" in "develop" later ?

**Comment by Mh-Asmi** [comment]
@tuxpiper 
if you want we can merge it into develop and add the comments to "release/2023.32" by cherry-pick

**Comment by tuxpiper** [comment]
ah yep that works @Mh-Asmi  , once we have cherry-picked once in this branch is not easy to merge in

**Pull request merged**